### DISCUSSION TITLE
How to use v-model in dynamic form

### DISCUSSION THREAD

**Discussion by alsa85** [quasarframework/quasar#17261] [General - Components / Directives / etc]
Hi guys, I ask to any expert if can help me to set component which use as global form. 
This component take an object named (generalObject) from a props.
The component which call the global form take a generalObject from Vuex.
The global form render the form ad assign v-model to any field (all based on data passed through an object that defines the logic).
With this setup the console give me the error because I try to update state outside the mutation, but I can't know which are the fields and I don't know also how many fields did form have, therefore I can't set a computed for any field which provide to get and set value as the best practice recommend. So is there a system that avoids me having to use the computed for each field?
I hope I explain all clear

**Comment by ivanjaros** [comment]
No, you cannot mutate props. Well, technically you can arrays and objects but that is merely due to technical limitations and in general you should use events to propagate changes upward. That's basic Vue stuff, not related to Quasar whatsoever.
